notebooks/08A_composing_parallel_sorting_data.ipynb

```diff
@@ -238,24 +250,26 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Total files before: 15734\n",
-      "Total files after: 12641\n"
+      "Total profiles before: 12916\n",
+      "Total profiles after: 10624\n"
      ]
     }
    ],
    "source": [
-    "print(f\"Total files before: {len(tk.profile)}\")\n",
+    "print(f\"Total profiles before: {len(tk.profile)}\")\n",
     "tk = tk.filter_metadata(lambda meta: all([meta[key] in META_WHITELIST_DICT[key] for key in META_WHITELIST_DICT.keys()]))\n",
-    "print(f\"Total files after: {len(tk.profile)}\")"
+    "print(f\"Total profiles after: {len(tk.profile)}\")"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# 2C. Filter Duplicate Metadata Values\n",
+    "### 3C. Filter Duplicate Metadata Values\n",
+    "\n",
+    "Duplicate values across all of our experiment parameters indicates that one profile has incorrect metadata, since all of the profiles are single-trial. This is typically user error (metadata is manually annotated in Adiak).\n",
```

Comment:
e038cd3d
Yes this is removing all profiles that have duplicated metadata, which is not a lot. For example we remove one set of 27 profiles and one set of 26. Since this is single trial data, only 1/27 and 1/26 profiles have correct metadata, so it is not a huge loss to lose 2 correctly annotated profiles.

> Of course this raises the question of was any of the data validated if it's all manually annotated? Hopefully we can trust that people did it right in all other 

The dataset is not 100% validated, which would require manually inspecting each implementation for correctness. Although we do take a stab at this with anomaly removal e.g. if someone annotated a value as 1024 procs but ran with 2 procs, it would be removed as a slow outlier, but this does not solve the problem completely.